apps/web/pages/success.tsx

```diff
@@ -263,29 +284,95 @@ export default function Success(props: inferSSRProps<typeof getServerSideProps>)
                         <h3
                           className="text-2xl font-semibold leading-6 text-neutral-900 dark:text-white"
                           id="modal-headline">
-                          {needsConfirmation ? t("submitted") : t("meeting_is_scheduled")}
+                          {needsConfirmation
+                            ? props.recurringBookings
+                              ? t("submitted_recurring")
+                              : t("submitted")
+                            : props.recurringBookings
+                            ? t("meeting_is_scheduled_recurring")
+                            : t("meeting_is_scheduled")}
                         </h3>
                         <div className="mt-3">
                           <p className="text-sm text-neutral-600 dark:text-gray-300">
-                            {needsConfirmation
+                            {props.recurringBookings
+                              ? needsConfirmation
+                                ? props.profile.name !== null
+                                  ? t("user_needs_to_confirm_or_reject_booking_recurring", {
+                                      user: props.profile.name,
+                                    })
+                                  : t("needs_to_be_confirmed_or_rejected_recurring")
+                                : t("emailed_you_and_attendees_recurring")
+                              : needsConfirmation
```

Comment:
Aren't we abusing ternaries here? Can we try using a function with early returns?